client/components/NewMeetup.js

```diff
@@ -2,20 +2,164 @@ import React, {Component} from 'react';
 import { connect } from 'react-redux';
 import {  } from '../store';
 import { Redirect } from 'react-router-dom';
+import * as _ from 'lodash';
+import axios from 'axios';
+import {daysInMonth} from '../../utils';
 
 class NewMeetup extends Component {
-    constructor(){
+    constructor(props){
         super();
+        let { user } = props,
```

Comment:
Why are you using a "let" here instead of a "const"?

Reply:
just being lazy =).
didn't want to type const {user} = props;
then let now = new Date();
etc. =)
but in all fairness you are right. I'll change it.